notifiers/webhook.go

```diff
@@ -104,13 +120,8 @@ func (w *webhooker) sendHttpWebhook(body string) (*http.Response, error) {
 			}
 		}
 	}
-	if w.ApiKey.String != "" {
-		req.Header.Add("Content-Type", w.ApiKey.String)
-	} else {
-		req.Header.Add("Content-Type", "application/json")
-	}
-	req.Header.Set("User-Agent", "Statping")
-	req.Header.Set("Statping-Version", utils.Params.GetString("VERSION"))
+
+	utils.Log.Infoln(fmt.Sprintf("sending body: '%v' to %v as a %v request with headers: '%v'", body, w.Host.String, w.Var1.String, req.Header))
```

Comment:
Will this leak ApiSecret into the logs?

Reply:
Yes it would be adding them to the logs. Mostly added because I wanted to be sure it was even being sent. Not super attached to the logging though we can get rid of it.